official/vision/beta/projects/yolo/ops/preprocessing_ops.py

```diff
@@ -0,0 +1,931 @@
+import tensorflow as tf
+import numpy as np
+import random
+import os
+
+import tensorflow_addons as tfa
+from official.vision.beta.ops import box_ops as bbox_ops
+
+PAD_VALUE = 114
+GLOBAL_SEED_SET = False
+
+
+def set_random_seeds(seed=0):
+  """Sets all accessible global seeds to properly apply randomization.
+
+  This is not the same as passing the seed as a variable to each call 
+  to tf.random.For more, see the documentation for tf.random on the tensorflow 
+  website https://www.tensorflow.org/api_docs/python/tf/random/set_seed. Note 
+  that passing the seed to each random number generator will not give you the 
+  expected behavior if you use more than one generator in a single function.
+
+  Args: 
+    seed: `Optional[int]` representing the seed you want to use.
+  """  
+  if seed is not None:
+    global GLOBAL_SEED_SET
+    os.environ['PYTHONHASHSEED'] = str(seed)
+    random.seed(seed)
+    GLOBAL_SEED_SET = True
+  tf.random.set_seed(seed)
+  np.random.seed(seed)
+
+
+def get_pad_value():
```

Comment:
Why do we need this? can we just retrieve the constant?

Reply:
i don't want it to get changed by accident. but i can just use it directly If needed.

Reply:
it is used outside of this file

Reply:
addressed